.github/workflows/test.yml

```diff
@@ -185,8 +205,12 @@ jobs:
         run: |
           . ./ci-cargo.ps1
           foreach($target in ($env:BUILD_TARGETS).Split(',')) {
+            $features = ${{ matrix.config.features }}
+            if (-eq $features "full-functionality") {
+               $testArgs += "--test-threads=1"
+            }
             # Note: no feature is specified, which means such features like graphics, serde, ndarray, and num-complex are not tested here.
-            ci-cargo test $(if($target -ne 'default') {"--target=$target"} ) '--' --nocapture -ActionName "Testing for $target target"
+            ci-cargo test --features ${{ matrix.config.features }} @( $testArgs ) $(if($target -ne 'default') {"--target=$target"} ) '--' --nocapture -ActionName "Testing for $target target"
```

Comment:
@CGMossa here is my attempt to add the `--test-threads=1` for full-functionality. I cannot figure out the incantation.